packetbeat/tests/system/test_0070_drda_simple.py

```diff
@@ -0,0 +1,13 @@
+from packetbeat import BaseTest
+
+class Test(BaseTest):
+
+    def test_simple(self):
+        self.render_config_template(
+            drda_ports=[1527],
+        )
+
+        self.run_packetbeat(pcap="drda/drda_simple.pcap", debug_selectors=["*"])
+        objs = self.read_output()
+
+        assert len(objs) == 21
```

Comment:
Would be nice :-). What we usually try to do with unit test is to check for conditions like "a message  split between to TCP segments", that are harder to get with system tests. See the mysql module for some examples.